### DISCUSSION TITLE
File preview on website doesn't match file if you download it

### DISCUSSION THREAD

**Discussion by zxclwnq** [community/community#112705] [Repositories]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

I have project [ML-task-01](https://github.com/zxclwnq/ML-task-01). The problem is in fact that if you open [task1_numpy_practive.ipynb](https://github.com/zxclwnq/ML-task-01/blob/main/task1_numpy_practive.ipynb) preview, you can see old version of this file, but if you download, you'll get latest version of file
**This is in preview**
![image](https://github.com/community/community/assets/98162381/17e45c3b-516e-4b8b-b17d-0d61cbe2d845)
**This is in file (after this code there is a picture and file continuation**
![image](https://github.com/community/community/assets/98162381/b8fec2cc-9634-4a92-b552-5c58be705429)
I'm pretty sure that all files are in folder of project and all edits are committed and pushed

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @zxclwnq ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
